### PR TITLE
silently validate yandex key and remove anything after the first space

### PR DESCRIPTION
**Pull request by rohankatyal29**
#106 

@Ceilican, this seemed like a much better idea

### CODE REVIEW

**Comment by ankit-m** [comment]
The code looks good.

File: Source/options.js [hunk 1]
```diff
@@ -513,13 +513,9 @@ $(function() {
...
   function validate_yandexTranslatorApiKey(){
-    var yandexKey = e("yandexTranslatorApiKey").value
-    if (yandexKey.match(/Unblock|Block/)){
-      alert("Please enter valid Yandex key.\nRemove Block key / Unblock key from the end of the key")
-    }
-    else {
-      save_yandexTranslatorApiKey()
-    }
+    var yandexKeyInputElement = e("yandexTranslatorApiKey")
+    yandexKeyInputElement.value = yandexKeyInputElement.value.split(' ')[0]
```
Comment (Ceilican) [inline Source/options.js]: It might be a good idea to add a comment in the code explaining why we are doing this.

**Comment by Ceilican** [comment]
Thanks, Rohan!

File: Source/options.js (hunk 1 above)
Comment (rohankatyal29) [inline Source/options.js]: Added! 

Will push along with the next commit

**Pull request merged**